src/plugins/discover/public/application/apps/main/components/doc_table/doc_table_embeddable.tsx

```diff
@@ -21,33 +21,55 @@ export interface DocTableEmbeddableProps extends DocTableProps {
   totalHitCount: number;
 }
 
+const scrollTop = () => {
```

Comment:
I can see a problem here, you know there could be more then 1 saved search embeddable displayed, than always the first one would scroll

Reply:
Thanks for catching another issue! Fixed with adding `forwardRef` usage.